client/my-sites/customer-home/style.scss

```diff
@@ -1,110 +1,174 @@
-.customer-home__layout-col {
-	@include breakpoint('>960px') {
-		width: 49%;
-	}
-}
+.customer-home {
+	&__box-action {
+		margin-bottom: 16px;
+		width: 100%;
+
+		.button {
+			border-width: 1px;
+			display: block;
+			min-height: 85px;
+			min-width: 100%;
+			padding: 24px;
+	    text-align: center;
+		}
+		.button:hover {
+			box-shadow: 0 2px 3px 0 rgba( 98, 109, 118, 0.15 );
+		}
+
+		span {
+			display: block;
+		}
 
-.customer-home__card-subheader {
-	color: var( --color-text-subtle );
-	margin: -0.5rem 0 1rem;
-    font-size: 0.85rem;
-}
+		@include breakpoint('>1280px') {
+			margin-bottom: 24px;
+			width: 50%;
 
-.customer-home__layout {
-	@include breakpoint('>960px') {
+			.button {
+		    margin: 0 12px;
+		    min-height: 105px;
+		    min-width: 90%;
+		    padding: 30px 24px;
```

Comment:
more indentation issues

Reply:
Thanks. Fixed in: https://github.com/Automattic/wp-calypso/commit/81e150b3376bee4572ded0f896b7bdf06b424fe0